### DISCUSSION TITLE
expense ansible backend

### DISCUSSION THREAD

**Discussion by palsachennaiah** [learndevopsonline/learndevopsonline#1681] [Q&A] [answered]
![Screenshot 2023-12-19 122204](https://github.com/learndevopsonline/learndevopsonline/assets/148023299/64ed322d-78c7-4068-9164-1ac0e9df7dc6)

**Comment by sreedharm07** [comment, marked as answer]
@palsachennaiah 
remove that it will work
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/d84e5399-fa20-4ad0-afcf-3fac8fd7d3bf)

**Reply by palsachennaiah** [reply]
ok it is working thank you.
